src/test/java/uk/gov/register/service/ItemConverterFunctionalTest.java

```diff
@@ -0,0 +1,107 @@
+package uk.gov.register.service;
+
+import com.fasterxml.jackson.databind.JsonNode;
+import io.dropwizard.jackson.Jackson;
+import org.junit.Before;
+import org.junit.Test;
+import uk.gov.register.RegisterConfiguration;
+import uk.gov.register.configuration.ConfigManager;
+import uk.gov.register.configuration.FieldsConfiguration;
+import uk.gov.register.configuration.RegisterConfigConfiguration;
+import uk.gov.register.configuration.RegisterDomainConfiguration;
+import uk.gov.register.core.FieldValue;
+import uk.gov.register.core.LinkValue;
+import uk.gov.register.core.ListValue;
+import uk.gov.register.core.StringValue;
+import uk.gov.register.exceptions.NoSuchConfigException;
+import uk.gov.register.resources.RequestContext;
+
+import java.io.IOException;
+import java.util.Map;
+import java.util.Optional;
+
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.Matchers.equalTo;
+import static org.hamcrest.Matchers.instanceOf;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.when;
+
+public class ItemConverterFunctionalTest {
```

Comment:
I  also don't think these are really functional tests so can you remove Functional from the name?